src/Lidar.cc

```diff
@@ -225,6 +225,26 @@ bool Lidar::PublishLidarScan(const ignition::common::Time &_now)
   return this->PublishLidarScan(math::secNsecToDuration(_now.sec, _now.nsec));
 }
 
+//////////////////////////////////////////////////
+void Lidar::ApplyNoise()
```

Comment:
This looks like there may have been a bad merge?  The `ApplyNoise` function is defined above on line 203, which is causing the CI problems.